capstone/src/main/java/com/google/sps/servlets/ServletHelper.java

```diff
@@ -63,4 +68,24 @@ public static Entity getEntityFromId(
       return null;
     }
   }
+
+  /**
+   * Retrieves the user entity from the database based on id.
+   *
+   * @param response HttpServletResponse
+   * @param id id of user
+   * @param datastore datastore holding all data
+   * @return Entity
+   * @throws IOException error thrown from sendError method
+   */
+  public static Entity getUserFromId(
+      HttpServletResponse response, String id, DatastoreService datastore)
+      throws IOException {
+    try {
+      return datastore.get(KeyFactory.createKey("User", id));
+    } catch (EntityNotFoundException e) {
+      ErrorHandler.sendError(response, "Cannot get entity from datastore");
+      return null;
+    }
+  }
```

Comment:
The only difference that I see between getUserFromId and getEntityFromId is that one uses a string id and the other uses a long. Rather than write a specific one for a User entity, I recommend either 
a) Converting the string id to a long and call getEntityFromId
b) Create an overloaded getEntityFromId method that takes in a string for the id but then calls the other getEntityFromId and converts the string id to a long.
If you need to use getUserFromId many times I recommend option b, otherwise a is fine as well

Reply:
I think the userId is larger than the maximum allowed for a long so I think we are gonna have to use BigInteger